src/writers/file_writer/mod.rs

```diff
@@ -0,0 +1,13 @@
+pub mod header_writer;
+pub mod http_get_writer;
+use crate::readers::assembler::Endpoint;
+
+pub fn write(endpoints: &Vec<&Endpoint>) -> std::io::Result<()> {
+    let writer = http_get_writer::HTTPGetEndpointBuilder::new();
+    for endpoint in endpoints {
+        let endpoint_string = writer.create_endpoint(endpoint);
+        // TODO: placeholder code obviously
+        println!("{}", endpoint_string);
+    }
+    Ok(())
+}
```

Comment:
This code is being worked on now and will be the subject of the next PR so skipping over this is fine (mostly refactored anyway).